src/app/input_gen.ml

```diff
@@ -0,0 +1,60 @@
+open Biokepi.EDSL.Library.Input
+open Nonstd
+module String = Sosa.Native_string
+
+let cmd () =
+  let open Cmdliner in
+  let version = "0.0.0" in
+  let doc = "Submit a PGV Pipeline" in
```

Comment:
Could take the version from `Biokepi_run_environment.Metadata.version`?

The `doc` argument is wrong :)